src/core/show-status.h

```diff
@@ -8,13 +8,15 @@
 /* Manager status */
 
 typedef enum ShowStatus {
-        _SHOW_STATUS_UNSET = -2,
-        SHOW_STATUS_AUTO = -1,
-        SHOW_STATUS_NO = 0,
-        SHOW_STATUS_YES = 1,
-        SHOW_STATUS_TEMPORARY = 2,
+        SHOW_STATUS_NO,
+        SHOW_STATUS_AUTO,
+        SHOW_STATUS_TEMPORARY,
+        SHOW_STATUS_YES,
+        _SHOW_STATUS_INVALID = -1,
```

Comment:
usually we also define _SHOW_STATUS_MAX here, and use it to size the string array, so that the string array is never sized too short should we forget one entry...

Reply:
btw, now that it is normalized, might be cool to add it to src/test/test-tables.c too!